### PR TITLE
{perf}[gompi/2021a] DistributedStream v1.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'DistributedStream'
+version = '20210628'
+local_commit = '998f747'
```
Comment (boegel) [inline easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb:5]: @adrianjhpc Do you have any plans to tag a "proper" versioned release for `DistributedStream`, or should we keep using a specific commit and resort to a date stamp as "version" (like I'm doing here)?

Comment (adrianjhpc) [reply easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb:5]: Let me do a proper version release.

Comment (adrianjhpc) [reply easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb:5]: Release version 1.0 should now be available. Let me know if there are changes  I could make to integrate better into easybuild. Currently it's just a rubbish makefile, so if that doesn't integrate well let me know.

Comment (boegel) [reply easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb:5]: :+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2684.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/bece6f8d98cc3e0910766df7bc685788 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/c483e20033d70c0c415b2c3eda245e59 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**